WalletWasabi/WebClients/Wasabi/IHttpClientFactory.cs

```diff
@@ -0,0 +1,10 @@
+using WalletWasabi.Tor.Http;
+using WalletWasabi.Tor.Socks5.Pool.Circuits;
+
+namespace WalletWasabi.WebClients.Wasabi
+{
+	public interface IBackendHttpClientFactory
```

Comment:
I'm not too sure about this as we have `ExternalHttpClientFactory` so `IHttpClientFactory` comes to mind first.

Reply:
My first idea was to have a delegate but we agreed with David to not use delegates but segregated interfaces.